packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -2896,3 +3025,270 @@ typedef SelectableRegionContextMenuBuilder = Widget Function(
   BuildContext context,
   SelectableRegionState selectableRegionState,
 );
+
+/// The status of the [SelectionListener].
+enum SelectionListenerStatus {
+  /// Indicates the selection under a [SelectionListener] is changing.
```

Comment:
> is changing

Should they expect the selection value to have already updated?

Reply:
Yes, the selection value should already be updated when the user receives the notification.